vgen/src/test/java/io/gapi/vgen/testdata/discoveries/nodejs/nodejs_sqladmin.v1beta4.json.baseline

```diff
@@ -1331,20 +1331,13 @@ authFactory.getApplicationDefault(function(err, authClient) {
     auth: authClient
   };
 
-
-  var recur = function(err, result) {
+  sqladmin.users.list(request, function(err, result) {
```

Comment:
This new result is correct. The NodeJS API seems to think this is not page streaming. It is rather unfortunate that the compilecheck didn't catch this. I'll open an issue.